waldo/app/controllers/HelloWorldController.js

```diff
@@ -0,0 +1,19 @@
+var blueprint = require ('@onehilltech/blueprint')
```

Comment:
We will want to get rid of this file, but it does not have to happen this PR _"pull request"_